source/User_Guide/Transactional_Templates/how_to_send_an_email_with_dynamic_transactional_templates.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: page
+weight: 100
+title: How to send an email with Dynamic Transactional Templates
+navigation:
+  show: false
+seo:
+  title: How to send an email with Dynamic Transactional Templates
+  override: true
+  description: 
+---
+
+- [Before you begin](#-Before-you-begin)
+- [Design a dynamic transactional template](#-Design-a-dynamic-transactional-template)
+- [Send a transactional email](#-Send-a-transactional-email)
+  - [Send email using the SendGrid API](#-Send-email-using-the-SendGrid-API)
+  - [Send email using SMTP](#-Send-email-using-SMTP)
+
+{% anchor h2 %}
+Before you begin
+{% endanchor %}
+
+Before you create and send an email using a dynamic transactional template you need to do the following:
+
+* Create a SendGrid account
+* Create an [API Key]({{root_url}}/User_Guide/Settings/api_keys.html)
+* Add an unsubscribe group (optional)
+
+{% anchor h2 %}
+Design a dynamic template
+{% endanchor %}
+
+1. Open the [transactional templates](https://sendgrid.com/dynamic_templates) page and click **Create Template**.
+1. Add a unique template name and then click **Save**. 
+1. To begin editing your new template, click **Add Version**.
+1. Select an editor and click **Continue**.
+1. Design your template. For more information on designing a Dynamic Transactional Template using the Design Editor or Code Editor, see [Create and edit Dynamic Transactional Templates]({{root_url}}/User_Guide/Transactional_Templates/create_and_edit_dynamic_transactional_templates.html).
+
+Here's a sample template: <a href="/source/assets/sample_dynamic_template.html" target="_blank">Hipster Cat Cafe</a>
```

Comment:
This download link is broken. Link out to examples in the using handlebars page instead of having this link